guide/17-geocoding-guides/part1_what_is_geocoding.ipynb

```diff
@@ -0,0 +1,1334 @@
+{
```

Comment:
<ul><li>Reverse geocode: The Geocoding Service can reverse geocode an x/y location and return the nearest address or place. Reverse geocoding is the process of converting a point to an address or place. For example, you can convert&nbsp;<code>-79.3871 longitude and 43.6426 latitude</code>&nbsp;to&nbsp;<code>CN Tower, 301 Front St W, Toronto, Ontario, M5V, CAN</code>. Generally speaking, you can use reverse geocoding to:</li><li class="ql-indent-1">Get the nearest address to your current location.</li><li class="ql-indent-1">Show an address or place name when you tap on a map.</li><li class="ql-indent-1">Find the address for a geographic location.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/830/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>